tests/system/pages/loginPage/WatchOnlyLoginPage.ts

```diff
@@ -1,33 +1,43 @@
+import { step } from 'allure-js-commons';
+
 import { BasePage } from '../BasePage';
 import { AssetsPageElements } from '../_elements/AssetsPageElements';
 import { type LoginPageElements as OnboardingPageElements } from '../_elements/LoginPageElements';
 import { WatchOnlyAssetsPage } from '../assetsPage/WatchOnlyAssetsPage';
 
 export class WatchOnlyOnboardingPage extends BasePage<OnboardingPageElements> {
   public async fillAccountAddress(address: string): Promise<WatchOnlyOnboardingPage> {
-    await this.click(this.pageElements.enterAccountAddress);
-    await this.page.getByTestId(this.pageElements.enterAccountAddress).fill(address);
+    await step(`Fill account address: ${address}`, async () => {
+      await this.click(this.pageElements.enterAccountAddress);
+      await this.page.getByTestId(this.pageElements.enterAccountAddress).fill(address);
+    });
 
     return this;
   }
 
   public async fillWalletName(name: string): Promise<WatchOnlyOnboardingPage> {
-    await this.page.getByTestId(this.pageElements.accountNameField).fill(name);
+    await step(`Fill wallet name: ${name}`, async () => {
+      await this.page.getByTestId(this.pageElements.accountNameField).fill(name);
+    });
 
     return this;
   }
 
   public async createWatchOnlyAccount(name: string, address: string): Promise<WatchOnlyAssetsPage> {
-    await this.fillWalletName(name);
-    await this.fillAccountAddress(address);
-    await this.click(this.pageElements.continueButton);
-    await this.page.waitForTimeout(5000); // takes some time to load the app and balances
-
-    return new WatchOnlyAssetsPage(this.page, new AssetsPageElements());
+    return await step(`Create Watch Only account [${name} | ${address}]`, async () => {
```

Comment:
Lets remove params here, in order to stick to DRY

<img width="545" alt="Screenshot 2025-06-26 at 18 11 25" src="https://github.com/user-attachments/assets/fbb7541f-d939-4c5d-8a2f-ed2d7dfc5c0f" />